src/routes/cartsRouter.js

```diff
@@ -5,10 +5,12 @@ const cartsRouter = express.Router();
 const {
   addItemToCartsController,
   getCartsController,
+  modifyQuantityController,
 } = require('../controllers/cartsController');
 const { validateAccessToken } = require('../middlewares/validateAccessToken');
 
 cartsRouter.get('/', validateAccessToken, getCartsController);
 cartsRouter.post('/', validateAccessToken, addItemToCartsController);
+cartsRouter.patch('/quantity', validateAccessToken, modifyQuantityController);
```

Comment:
```suggestion
cartsRouter.patch('/:cartId', validateAccessToken, modifyQuantityController);
```
특정 장바구니 데이터를 변경하는 것이기 때문에 body에 quantity만 받고 query param으로 cartId를 받는 것은 어떨까요?

```JavaScript
// cart 수정 API

// 👍 - 한 눈에 1번 카트를 수정하는 것을 알 수 있음
PATCH :3000/carts/1
```